language-design.md

```diff
@@ -48,7 +72,9 @@ How do the themes of _Growing a Language_ relate to the lab we did this week?
 In what way is an API a language? 
 
 **Response**
+An API is a languge in that it has a predefined vocabulary and syntax. -Daniel
 
+I don't think API's typically have syntax, although I guess chaining a few methods together could be part of an API and is kind of like syntax. In that case an API might resemble an internal DSL but still have the host language's flavor. In any case, as far as I understand, APIs are just the method (or function) names for a library, and Steele says "[a] true library does not change the rules of meaning for the language; it just adds new words." [Steele 1998] -Aaron
 
```

Comment:
I agree with Aaron in that an API doesn't necessarily require a predefined syntax. I think it would be more accurate to describe it as a predefined vocabulary and rules (perhaps this is what Daniel meant), which is basically a language, according to Steele. It would be nice for more explanation and unpacking the Steele quote a little more.